scripts/ci/github_actions/driver/setup_ci_dependencies.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -xe
+set +o pipefail
+
+python -m pip install --upgrade pip docker-hub==2.2.0
+echo docker-hub > dev-requirements.txt
+
+cat >>/home/runner/.bash_profile <<'EOL'
+yq() {
+  docker run --rm -e operator_image_for_test=$operator_image_for_test\
+                  -e cr_image_value=$cr_image_value\
```

Comment:
where is `cr_image_value` coming from?

Reply:
removed for now the `yq` function at all because I am not using it anymore in the script

Reply:
then how does [this](https://github.com/IBM/ibm-block-csi-driver/blob/1c9a227fb0c26106d96307a982563943d8c8d446/scripts/ci/github_actions/driver/setup_ci_dependencies.sh#L10) work?

Reply:
yes you're right, my bad, moved it back and removed the `-e`